src/index.js

```diff
@@ -0,0 +1,130 @@
+// Constants
+const BASE_URL = 'http://127.0.0.1:5000';
+const DEFAULT_CITY = 'Los Angeles';
+const increaseTemp = document.getElementById('increaseTempControl');
+const decreaseTemp = document.getElementById('decreaseTempControl');
+const tempValue = document.getElementById('tempValue');
+const landscape = document.getElementById('landscape');
+const cityNameInput = document.getElementById('cityNameInput');
+const cityNameReset = document.getElementById('cityNameReset'); 
+const headerCityName = document.getElementById('headerCityName');
+const currentTempButton = document.getElementById('currentTempButton');
+
+// Changes color + landscape by Temperature
+let temp = 60;
+const changeTemp = () => {
+    let landscapeEmojis = '';
+    if (temp >= 80) {
+        tempValue.style.color = 'red';
+        landscapeEmojis = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    } else if (temp< 80 && temp >= 70) {
+        tempValue.style.color = 'orange';
+        landscapeEmojis = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (temp < 70 && temp >= 60) {
+        tempValue.style.color = 'yellow';
+        landscapeEmojis = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else if (temp < 60 && temp >= 50) {
+        tempValue.style.color = 'green';
+        landscapeEmojis = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    } else {
+        tempValue.style.color = 'teal';
+        landscapeEmojis = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    }
+    landscape.textContent = landscapeEmojis;
+};
+
+// Changes sky by selection
+const changeSky = () => {
+    const selectedSky = document.getElementById('skySelect').value;
+    const theSky = document.getElementById('sky');
+    let sky = '🌞😎🌞';
+    if (selectedSky === 'sunny') {
+        sky = '🌞😎🌞';
+    } else if (selectedSky === 'cloudy') {
+        sky = '🌥️🌥️🌥️';
+    } else if (selectedSky === 'rainy') {
+        sky = '🌧️🌧️🌧️';
+    } else if (selectedSky === 'snowy') {
+        sky = '🌨️🌨️🌨️';
+    }
```

Comment:
Consider building a dictionary where the key is the sky setting, and the values hold the emoji string and display class. Then rather than writing another conditional chain, we could look up the values directly in the dictionary, simplifying the code and making it more data driven. We could even enumerate the keys and use them to dynamically build the options in the sky selection!